users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -258,18 +141,18 @@ def get_masked():
         time_mask_max_num = tf.gather(specaug_params["time_mask_max_num"], current_epoch)
         freq_mask_max_num = tf.gather(specaug_params["freq_mask_max_num"], current_epoch)
         freq_mask_max_size = tf.gather(specaug_params["freq_mask_max_size"], current_epoch)
-        time_mask_max_proportion = tf.cast(
+        total_time_masks_max_size = tf.cast(
```

Comment:
```suggestion
        total_time_masks_max_frames = tf.cast(
```
Wouldn't this reflect it better?